### PR TITLE
UIH-619: Create-react-app with React Storybook Demo

### PR DESCRIPTION
**Pull request by kobomarun**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "tuts",
+  "version": "0.0.1",
+  "private": true,
```
Comment (jabdul) [inline package.json]: Why is this private? We should perhaps remove this.

File: src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+});
```
Comment (jabdul) [inline src/App.test.js:41]: There's no assertion in this test, so what's the purpose?
 I suggest add a meaningful test

File: src/components/Counter/Counter.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { css } from 'aphrodite';
+import CounterStyles from './Counter.styles'
+
+const Counter = ({ onClick, children }) => {
+  return(
+    <div>
+    <button
```
Comment (jabdul) [inline src/components/Counter/Counter.jsx:7]: Indent

File: src/components/Counter/Counter.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { css } from 'aphrodite';
+import CounterStyles from './Counter.styles'
```
Comment (jabdul) [inline src/components/Counter/Counter.jsx]: Rename `CounterStyles` to `styles`. 

Less verbose

File: src/components/Counter/Counter.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  );
+}
+
+Counter.defaultProps = {
+  onClick: () => void(0),
+  children: 'Click to count'
+}
+
+Counter.PropTypes = {
+  onClick: React.PropTypes.Func
```
Comment (jabdul) [inline src/components/Counter/Counter.jsx]: What about `children`?

File: src/components/Counter/Counter.styles.jsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+const CounterStyles = StyleSheet.create({
+  button: {
+    margin: '10px',
+    backgroundColor: '#ff0000',
+    color: '#fff',
+    fontSize: '16px',
+    width: '100px',
+    height: '40px',
+    border: 'none',
+    cursor: 'poiter',
```
Comment (jabdul) [inline src/components/Counter/Counter.styles.jsx]: `pointer`

File: src/components/Counter/MainCounter.jsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  }
+
+
+  render() {
+    return(
+      <div>
+        <p>{this.state.count}</p>
+      <Counter onClick={this.handleClickIncrement}>
+        {'Increment'}
+      </Counter>
```
Comment (jabdul) [inline src/components/Counter/MainCounter.jsx]: Indent

**Review by jabdul — CHANGES_REQUESTED** [review]

File: src/components/Counter/Counter.styles.jsx [hunk 8]
```diff
@@ -0,0 +1,16 @@
+import { StyleSheet } from 'aphrodite';
+
+const CounterStyles = StyleSheet.create({
+  button: {
+    margin: '10px',
+    backgroundColor: '#ff0000',
+    color: '#fff',
+    fontSize: '16px',
+    width: '100px',
+    height: '40px',
```
Comment (jabdul) [inline src/components/Counter/Counter.styles.jsx:10]: By default, constant values are coerced into `px` units.

So
```css
height: '40px'
```

is same as...

```css
height: 40
```

Comment (kobomarun) [reply src/components/Counter/Counter.styles.jsx:10]: This did not work with `px`

**Pull request merged**